lib/pbench/server/utils.py

```diff
@@ -93,21 +93,47 @@ def quarantine(dest, logger, *files):
             sys.exit(102)
 
 
-class IsoTimeHelper:
+class UtcTimeHelper:
+    """
+    A helper class to work with UTC "aware" datetime objects. A "naive" object
+    (without timezone offset) will be set to UTC timezone.
+    """
+
     def __init__(self, time: datetime.datetime):
+        """
+        Capture a datetime object and; if it's "naive", set it to be UTC; if
```

Comment:
s/and;/and,/

Reply:
I was trying to separate the two comma clauses: admittedly, I did this incorrectly as I should have used ":" instead of "and;" to set it up. I must have been tired.